dusty/commands/logs.py

```diff
@@ -3,7 +3,7 @@
 from ..systems.docker import get_dusty_containers
 from . import utils
 
-def tail_container_logs(app_or_service_name, follow=False, lines=None):
+def tail_container_logs(app_or_service_name, follow=False, lines=None, timestamps=True):
```

Comment:
There's a disconnect here between the CLI and this function (which is only called by the CLI). In the CLI itself, timestamps default to off (which, now that I think about it, should be how it works to mimic Docker itself), but in this function they're defaulted to True.

Think we just need to default this to False here then adjust the tests back.